libs/category/src/drivers/magento/transformers/applied-filter-transformer.service.ts

```diff
@@ -16,9 +16,18 @@ export class DaffMagentoAppliedFiltersTransformService {
 		};
 
 		daffFilters.forEach(filter => {
-			magentoFilters[filter.name] = {
-				...magentoFilters[filter.name],
-				[this.transformActionEnum(filter.action)]: filter.value
+			if(filter.action === DaffCategoryFilterActionEnum.FromTo) {
+				const fromToValues = filter.value.split('-');
```

Comment:
Should this be referencing `DaffCategoryFromToFilterSeparator` for the `split` argument?

Reply:
yea, definitely. Changed